group_project_v2/mixins.py

```diff
@@ -222,3 +230,26 @@ def studio_view(self, context):  # pylint: disable=unused-argument, no-self-use
         fragment = Fragment()
         fragment.add_content(NO_EDITABLE_SETTINGS)
         return fragment
+
+
+class DashboardMixin(object):
+    def dashboard_view(self, context):
```

Comment:
Perhaps this should be an [abstract method](https://docs.python.org/2/library/abc.html#abc.abstractmethod), then?